docs/arguments.md

```diff
@@ -75,7 +75,17 @@ Docker daemon socket to connect to. Can be pointed at a remote Docker host by sp
 Environment Variable: DOCKER_HOST
                 Type: String
              Default: "unix:///var/run/docker.sock"
-```      
+```
+
+## Docker API version
```

Comment:
This needs to include a note that we can't guarantee stability on any docker version earlier than 1.24.

Reply:
Added a note for minimum supported version.